hyftest.js

```diff
@@ -0,0 +1,60 @@
+// 1  Programming Basics
+const movies = ["The Shawshank Redemption",
+"The Godfather",
+"The Dark Knight",
+"12 Angry Men",
+"The Lord Of The Rings",
+"Pulp Fiction",
+"The Good, the Bad and the Ugly",
+"Forrest Gump"];
+
+//1. Log out each movie title;
+console.log(movies)
+
+//2. Log out the number of words in each movie title.
+movies.forEach(function(movies) {
+console.log(`Numbers of words in the ${movies}" is ${movies.split(" ").length}.`);
+   });
+   
+
+//2 DOM manipulation
+
+var button = document.createElement("button");
```

Comment:
Every time you use `var` instead of `let` or `const`, a puppy dies, as we have discussed in the first ever JS class. 🥺